jailer_test.go

```diff
@@ -14,11 +14,21 @@ package firecracker
 
 import (
 	"context"
+	"fmt"
+	"io/ioutil"
 	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
 )
 
+func getNumaCpuset(node int) string {
```

Comment:
How about having this function on jailer.go side? Args() is doing basically the same thing.

Reply:
done.